application/src/main/java/org/thingsboard/server/service/queue/processing/TbRuleEngineProcessingStrategyFactory.java

```diff
@@ -108,6 +114,11 @@ public TbRuleEngineProcessingDecision analyze(TbRuleEngineProcessingResult resul
                         } catch (InterruptedException e) {
                             throw new RuntimeException(e);
                         }
+                        if (multiplyPauseBetweenRetries && maxPauseBetweenRetries > 0) {
+                            if (pauseBetweenRetries != maxPauseBetweenRetries) {
+                                pauseBetweenRetries = Math.min(maxPauseBetweenRetries, pauseBetweenRetries * pauseBetweenRetries);
```

Comment:
Maybe 
`pauseBetweenRetries = Math.min(maxPauseBetweenRetries, pauseBetweenRetries * 2);`
?

Reply:
@ashvayka sorry for this typo.